ext/intl/config.m4

```diff
@@ -6,9 +6,25 @@ PHP_ARG_ENABLE(intl, whether to enable internationalization support,
 [  --enable-intl           Enable internationalization support])
 
 if test "$PHP_INTL" != "no"; then
-  PHP_SETUP_ICU(INTL_SHARED_LIBADD)
+  PKG_CHECK_MODULES([ICU], [icu-io >= 50.1],
+                    [ICU_FOUND=true], [ICU_FOUND=false])
+
+  if test "$ICU_FOUND" = false; then
+    AC_MSG_ERROR([icu version 50.1 or later is required for internationalization support.])
+  fi
+
+  PHP_EVAL_INCLINE($ICU_CFLAGS)
+  PHP_EVAL_LIBLINE($ICU_LIBS, $1)
```

Comment:
What's `$1` here?

Reply:
Should probably be `INTL_SHARED_LIBADD`?

Reply:
Yes, you're right. A copy/paste error on my part. Will fix now.